pyproject.toml

```diff
@@ -24,7 +24,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.dependencies]
 python = "^3.7"
-flake8 = ">=3.8.4,<6"
+flake8 = ">=3.8.4,<7"
```

Comment:
I'd suggest removing the upper constraint altogether (see https://github.com/rocioar/flake8-django/issues/104).